index.js

```diff
@@ -21,6 +21,13 @@ const forbiddenModules = [
 
 module.exports = {
   parser: 'babel-eslint',
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: 'module',
+    ecmaFeatures: {
```

Comment:
```suggestion
    ecmaFeatures: {
      impliedStrict: true,
```

Reply:
What would this imply then? I know we removed it from the app itself didn't we?

Reply:
Didn't we disable strict mode for One App v5? 🤔

Reply:
correct we did, probably does default to the purpose putting back in the linter